markdown/org/docs/patterns/shelly/options/straightsides/en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Straight sides"
+---
+
+Forms a tubular shirt, which is a shirt where the hips and chests are the same width and connected by a straight seam. Set this to true for a bodice that is perfectly rectangular below the armholes. This setting overrides any values set for the hips or body shape. For most body types, leaving this option on will result in a reasonable fit that is looser around the belly and tighter around the chest, and generally produces a more fabric-efficient pattern.
```

Comment:
```suggestion
Forms a tubular shirt, which is a shirt where the hips and chest are the same width and connected by a straight seam. Set this to true for a bodice that is perfectly rectangular below the armholes. This setting overrides any values set for the `hips ease` or `side shape` options. For most body types, leaving this option on will result in a reasonable fit that is looser around the belly and tighter around the chest, and generally produces a more fabric-efficient pattern.
```

Reply:
(Actually, I am not sure whether the option name is `side shape` or `body shape`. It is called by one name in one place and by the other name in in another place.)